src/containers/app/AppNavigationSidebar.js

```diff
@@ -22,13 +22,13 @@ import { STATE } from '../../utils/constants/StateConstants';
 
 const { NEUTRAL, PURPLE } = Colors;
 
-const DEFAULT_PADDING = css`padding: 20px 24px;`;
+const DEFAULT_PADDING = css` padding: 20px 24px; `;
```

Comment:
Are these spaces between back ticks actually triggering the linter o.O

Reply:
yeah, they either want new lines or spaces. linter is seeing it as "{<attribute>:<value>}" rather than "{ <attribute>:<value> }".